book/src/reference.md

```diff
@@ -15,6 +15,10 @@ on_failure = "bail"
 [preprocessor.admonish.default]
 collapsible = true
 
+[preprocessor.admonish.default.builtins_collapsible]
+summary = false
+warning = false
+
```

Comment:
I think this configuration is not very clear to the end user. I think we should have a new config section for builtin directives that's extensible in the future - I will rework to this now.

Reply:
Agreed, wasn't sure about the best way to do it for builtin directives...